feature/src/main/java/org/woowatechcamp/banchan/feature/home/HomeActivity.kt

```diff
@@ -117,22 +131,22 @@ class HomeActivity : AppCompatActivity() {
         )
     }
 
-    private fun getOtherDishFragment(tab: String): Fragment {
-        homeFragmentFactory.setTabType(tab)
+    private fun getTabFragment(tab: String?, fragmentName: String): Fragment {
```

Comment:
```suggestion
    private fun getTabFragment(tab: String? = null, fragmentName: String): Fragment {
```
로 바꾸면 위에 Best랑 Main은 더 간결하게 갖고올 수 있을 것 같네요!

Reply:
좋네요 반영하겠습니다!!